README.md

```diff
@@ -302,6 +302,60 @@ The list of objects consist of PR names and refs (either a branch or a sha) for
 1. Resolve with the list of objects consists of PR names and refs
 2. Reject if the input or output is not valid
 
+### getPrInfo
+The parameters required are:
+
+| Parameter        | Type  | Required | Description |
+| :-------------   | :---- | :------- | :-------------|
+| config        | Object | true | Configuration Object |
+| config.scmUri | String | true | The scm uri (ex: `github.com:1234:branchName`) |
+| config.token | String | true | Access token for scm |
+| config.prNum | Integer | true | The PR number used to fetch the PR |
+
+#### Expected Outcome
+The object consists of PR name, sha and ref for the pipeline.
+
+#### Expected Promise Response
+1. Resolve with the object consists of PR name, sha and ref
+2. Reject if the input or output is not valid
+
+### getScmContexts
+No parameters are required.
+
+#### Expected Outcome
+The array of scm context name (e.g. [github.com, my-gitlab])
+
+#### Expected Response
+1. The array of scm context name
+
+### canHandleWebhook
+The parameters required are:
+
+| Parameter        | Type  |  Description |
+| :-------------   | :---- | :-------------|
+| headers        | Object | The request headers associated with the webhook payload |
+| payload        | Object | The webhook payload received from the SCM service |
+
+#### Expected Outcome
+The received webhook is available or not as boolean.
+
+#### Expected Promise Response
+1. Resolve with the received webhook is available or not.
+2. Reject if the input or output is not valid
+
+### getDisplayName (overriding needs only the case of `scm-router`)
```

Comment:
`scm-github` and other scm modules use getDisplayName() as function of `scm-base` . `scm-router` only overrides this function

> Is scmContext only used for scm-router? Could it be an optional parameter?
 
Yes. We do not need scmContext and scmContext is ignored if it's added when we use single scm module. 
But we want to call the same interface which is `getDisplayName(scmContext)` in code of screwdriver API so we added the comment.

Multiple module case:
UI -> API -> getDisplayName(scmContext) of `scm-router` -> getDisplayName()  of `scm-base` of a specified module.

Single module case:
UI -> API -> getDisplayName(scmContext) of a `scm-module` -> getDisplayName()  of `scm-base` of the `scm-module`.